### PR TITLE
so/homepage version  2.1

### LINKED ISSUE #60: Homepage: Version 2.0
- Refactor the code keeping the original design with updated fonts

### PR DESCRIPTION
**Pull request by sssoja**
### What changes have you made?

- Refactored the homepage in line with design changes 

**Nb.** Design is mobile-first but I think there's something funny going on with the view height on desktop. Is anyone able to look at this with me? If I get to it first, will just update this PR once I've figured it out (avec fresh eyes) 

### Which issue(s) does this PR fix?

Fixes #60 

### Screenshots (if there are design changes)

Screenshot 2020-07-19 at 22 00 22

Screenshot 2020-07-19 at 22 00 32

### How to test
`yarn start`

### CODE REVIEW

File: src/components/BuyButton.tsx [hunk 1]
```diff
@@ -3,40 +3,27 @@ import { loadStripe } from "@stripe/stripe-js";
...
-  ColorProps &
-  SpaceProps &
-  TypographyProps &
-  BorderProps;
-
-const Button = styled.button<LogoProps>`
+const Button = styled.button<
+  SpaceProps & TypographyProps & BorderProps & BackgroundProps
+>`
+  width: max-content;
```
Comment (mantagen) [inline src/components/BuyButton.tsx:22]: Doesn't work on ie11

What problem was this fixing?

Could use: 
```
width: auto;
white-space: nowrap;
```

Orrrr poss better, `display: flex`?

Comment (sssoja) [reply src/components/BuyButton.tsx:22]: Basically I was trying to style everything that is seen on the home page including the buy button component - granted can understanding why this is confusing

Comment (sssoja) [reply src/components/BuyButton.tsx:22]: The reason for no `display: flex` was this

Screenshot 2020-07-20 at 19 16 04

but `width: auto;
white-space: nowrap;` is a winner

**Comment by sssoja** [comment]
Closing to create new branch with `develop` updates

**Pull request closed without merging**